pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "embryo"
```

Comment:
The name "embryo" is way to vague / all-encompassing. A better code name for us is epipeODE as in an ODE model of EPI-PE segregation.